links.opam

```diff
@@ -64,4 +64,5 @@ depends: [
   "cohttp"
   "websocket-lwt"
   "safepass"
+  "ocamlfind"
```

Comment:
I think it is unnecessary to list `ocamlfind` as it is bundled with the compiler.

Reply:
Are you sure? You can use ocamlfind as a frontned for the ocaml compiler, but it doesn't seem to be included in it. In particular it's available via opam:
https://opam.ocaml.org/packages/ocamlfind/